src/components/CharacterInfoMeter/CharacterInfoMeter.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file A custom element that represents a value with a minimum and a maximum.
+ */
+import BaseElement from "../_BaseElement";
+// @ts-expect-error
+import styles from "./CharacterInfoMeter.css" assert { type: "css" };
+import { component } from "../decorators";
+
+@component('uft-character-info-meter')
+export default class CharacterInfoMeter extends BaseElement {
+  static get styles() {
+    return [styles];
+  }
+
+  get #value() {
+    const value = parseInt(this.getAttribute("value") || '');
+    if (isNaN(value)) return 0;
+    return value;
+  }
+
+  get #max() {
+    const max = parseInt(this.getAttribute("max") || '');
+    if (isNaN(max)) return 100;
+    return max;
+  }
+
+  get #progress() {
+    return (this.#value / this.#max * 100) || 0;
+  }
```

Comment:
it's missing the (admittedly hard to make extensible) tick marks for each of the encumbrance step-down points

<img width="589" alt="image" src="https://github.com/vttred/ose/assets/1410433/793620f1-4d8e-476a-a387-305199dadc64">

Reply:
I like how it looks though